config/routes.rb

```diff
@@ -17,7 +17,9 @@
   post "/remove_item" => "cart_items#destroy"
   post "/update_item" => "cart_items#update"
 
-  patch "/categories/:id" => "categories#update"
-  resources :categories, only: [:create, :edit, :destroy, :new, :index]
+  scope 'admin', module: 'admin' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.